Code-4-Community/speak-for-the-trees-frontend
# Show stewardship activities per month

## Why

[ClickUp Ticket](https://app.clickup.com/t/1dk45bd)


On the tree page, remove the limit on the number of stewardship activities shown and show stewardship activities in a specified month, where the default month is the current month.

## This PR


Added a dropdown menu that the user can use to show stewardship activities in the selected month.
Added a pagination component below the stewardship activity list (which appears only if there are more than 10 stewardship activities that were recorded during the month selected by the user) to reduce the amount of scrolling.

## Screenshots

At most 10 activities shown:
![first](https://user-images.githubusercontent.com/33704204/138531999-633fb1c8-2d07-4db3-a4ff-f2740225e1bf.png)

More than 10 activities shown:
![second](https://user-images.githubusercontent.com/33704204/138532008-4a63997e-badc-434d-9c00-6a8901cba725.png)

 

## Verification Steps

 
Visual verification that the planting instruction message displayed is correct on Firefox. Ran `npm run check` and `npm run test`.
